types/vis/index.d.ts

```diff
@@ -1703,6 +1703,33 @@ export interface Edge {
   id?: IdType;
 }
 
+export interface Locales {
+    [language: string]: LocaleMessages | undefined;
+    en?: LocaleMessages;
+    de?: LocaleMessages;
+    es?: LocaleMessages;
+    it?: LocaleMessages;
+    nl?: LocaleMessages;
+    'pt-br'?: LocaleMessages;
+    ru?: LocaleMessages;
+}
+
+export interface LocaleMessages {
+    edit?: string;
+    del?: string;
+    back?: string;
+    addNode?: string;
+    addEdge?: string;
+    editNode?: string;
+    editEdge?: string;
+    addDescription?: string;
+    edgeDescription?: string;
+    editEdgeDescription?: string;
+    createEdgeError?: string;
+    deleteClusterError?: string;
+    editClusterError?: string;
```

Comment:
According to here: http://visjs.org/docs/network/index.html?#locales you're right, it's mandatory